common/summon/secrets.yml

```diff
@@ -1,10 +1,6 @@
 common:
   KUBECTL_CLI_URL: https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl
   OPENSHIFT_CLI_URL: https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
-  POSTGRES_DATABASE: !var qa/xa/p2p/postgresql/database
-  POSTGRES_HOSTNAME: !var qa/xa/p2p/postgresql/hostname
-  POSTGRES_USERNAME: !var qa/xa/p2p/postgresql/username
-  POSTGRES_PASSWORD: !var qa/xa/p2p/postgresql/password
```

Comment:
where were these used?

Reply:
they were used for the demo but I will remove the whole demo

Reply:
removed